onnxscript/_framework_apis/torch_2_5.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Stable APIs for PyTorch 2.5."""
```

Comment:
So if we move forward to 2.6, we generate a new file like this one?

Reply:
Likely, yes. It may simply be aliasing the existing functions